test/controllers/api/v2/errata_controller_test.rb

```diff
@@ -10,6 +10,7 @@ def self.before_suite
     def models
       ::Katello::Product.any_instance.stubs(:as_json).returns([])
       @repo = Repository.find(katello_repositories(:rhel_6_x86_64))
+      @errata_filter = ContentViewFilter.find(katello_content_view_filters(:populated_erratum_filter))
```

Comment:
you can actually get rid of that ContentViewFilter.find() and just use:
```@errata_filter = katello_content_view_filters(:populated_erratum_filter))```

We used to have to do that, but not anymore